components/steps/Profile.tsx

```diff
@@ -115,6 +112,9 @@ const ProfileStep: React.FunctionComponent<Props> = props => {
     <Flex direction="column">
       <Form
         onSubmit={e => {
+          if (userResume) {
+            uploadResume(userResume.files[0]);
```

Comment:
small thing: is it possible that syncProfile will finish before uploadResume given the async nature of these methods? `syncProfile` will end up redirecting to the results page so just wanna make sure resume upload is complete before that happens

Reply:
Oh yeah, thats a good call

Reply:
Wait, does the event Loop deschedule this if we redirect? We're not really waiting on anything to return since all the changes should happen on the server, so as long as it gets pushed to completion *eventually* we should be good

Reply:
hmmm honestly it might be good, i'll test on that heroku env w/ a simulated slow env

Reply:
kk